EIPS/eip-7748.md

```diff
@@ -193,11 +193,11 @@ def state_convert(state: State, stride: int):
                 else:
                     # No more storage-slots. Move to the code migration starting
                     # at chunk-number zero.
-                    state.conversion_curr_account.phase = CodePhase(0)
+                    state.conversion_curr_account.phase = BasicDataPhase()
```

Comment:
We've pending in this PR to decide a better name for `BasicDataPhase`.

Reply:
The best name I can think of is `AccountDataPhase`. It is not perfect as an account can be interpreted as including storage, but in verkle they are conceptually more separate than in the MPT since storage isn't nested, and is (mostly) distributed throughout the tree.